### ISSUE TITLE
Unable to Login or Submit Text - Ubuntu 24.10

### ISSUE THREAD

**Issue by bryan977** [OrangeDrangon/android-messages-desktop#476]
My system dual boots Ubuntu 24.04 and Ubuntu 24.10. On the 24.10 installation, this application will allow me to enter my Google Account; however, when I click the "Next" button, the "Email or phone" field appears blank and nothing else happens. When clicking in the "Email or phone" field, my email address re-appears; however again, when clicking "Next", nothing happens.

I have tried purging and reinstalling the app, along with deleting the ~/.config/android-messages-desktop" directory.

**Comment by theryanmc** [comment]
Having same issue in Windows.

**Comment by PXAbstraction** [comment]
Same issue here on Windows, but I was able to use QR code pairing successfully.

**Comment by focuspulling** [comment]
I'm having this problem too on Windows 11 with the latest version of the app.  Besides not being able to click or submit anything at the credentials screen, QR code pairing is also not possible because my account is associated with my e-mail address.

**Comment by OrangeDrangon** [comment]
I am looking into it

**Comment by OrangeDrangon** [comment]
I was able to login by selecting remove account. Removing my account. Signing in. Closing the app. Reopening the app.

**Comment by focuspulling** [comment]
I don't have that option anywhere: this app, my Android app, or the Web.

I think the giveaway to the problem is that clicking anywhere on the
credentials page triggers nothing.  You can't even select the popup to
choose another account.

**Comment by OrangeDrangon** [comment]
![image](https://github.com/user-attachments/assets/182dcde9-f055-4601-9976-1218756543b4)

**Comment by focuspulling** [comment]
This is not addressing the bug, which (as I noted earlier) prevents clicking the popup to even choose or remove a Google account. You might be trying to say (incompletely) that the temporary workaround is to log completely out of all Google accounts in the Windows Chrome browser, so as to log back in.  But the app bug persists.

**Comment by GTR8000** [comment]
> I was able to login by selecting remove account. Removing my account. Signing in. Closing the app. Reopening the app.

Nope, still no good in Windows. In fact I did a **complete** uninstall by manually deleting the various directories that were left behind after running the uninstaller (AppData\Local\android-messages-desktop-updater, AppData\Local\Programs\Android Messages, AppData\Roaming\android-messages-desktop) which wiped out all settings, and after reinstalling 5.6.0 fresh it wouldn't accept my email address. The Next button is completely ineffective. Pressing the enter key on the keyboard resulted in a 400 error "The server cannot process the request because it is malformed. It should not be retried."

I first noticed this a few days ago when attempting to re-pair an install where it was ignoring my password. Copy/paste didn't work nor did typing it directly. The "show password" option also did nothing, displaying asterisks in both cases.

Another complete uninstall followed by a clean install/revert to 5.5.0 fixed the issue on two Win 11 computers, so something is definitely borked up with 5.6.0

**Comment by OrangeDrangon** [comment]
should be working in latest i think

**Comment by bryan977** [comment]
So far so good!

**Comment by focuspulling** [comment]
I'm feeling optimistic, working so far
